tests/test_counting_hash.py

```diff
@@ -1071,3 +1076,41 @@ def test_find_all_tags_list_error():
         assert False, "a ValueError should be raised for incorrect k-mer size"
     except ValueError:
         pass
+
+
+def test_abundance_distribution_gzipped_bigcount():
+    infile = utils.get_temp_filename('test.fa')
+    in_dir = os.path.dirname(infile)
+    shutil.copyfile(utils.get_test_data('test-abund-read-2.fa'), infile)
+    outfile = utils.get_temp_filename('test_ct.gz')
+    script = scriptpath('load-into-counting.py')
+    args = ['-x', str(1e7), '-N', str(2), '-k', str(2)]
```

Comment:
these next few lines could be simplified to:

args = [ '-x', ..., htfile, infile])

without using the args.extend.